src/user/model/user.model.ts

```diff
@@ -4,29 +4,39 @@ import { Document, Schema } from 'mongoose';
 export enum UserRole {
     USER,
     ADMIN,
+    SUPER_USER,
```

Comment:
Wat is het verschil tussen ADMIN en SUPER_USER? Waarom dit onderscheid?

Reply:
SUPER_USER kan je niet via de GUI worden, en met SUPER_USER kan je checkpoints resetten via de API

Reply:
Aha. Een technisch beheerder van de applicatie, dus? Tja ... lastig om al deze rollen in één lijstje te hebben, niet?

Reply:
Yes idd, eens dat dat beter kan, maar voor nu werkbaar